### DISCUSSION TITLE
doksi hibás signature példa

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#52] [Kérdések] [answered]
A doksi szerint a `7eae9ecf-f735-4a4f-aa49-e85ea411a3132024051012384726549118-0ddc-4e30-81bc-eaddd6f54b21FA12BC4567CA12BC4588` stringből a `2e81f124c0ee66be1e4cca1af72eb198b1a1c02ad1dffa0943a4fa8db0e440e8` aláírókulcs állítódik elő.

Ez nem helyes, a fenti stringből a `E4QX2M3PBMVM1CYCHMXTPPWCU6CJ6J66J+JJ6RBZFKU=` aláírókulcs állítódik elő. A példa nem is lehetne helyes, nincs nagybetűsítve.

**Comment by TamasPill** [comment, marked as answer]
Köszönjük a jelzést, javítva lett.
A nagybetűsítést csak a legvégén, a már base64 konvertált állapoton kell csinálni
